Microsoft.Toolkit.Uwp.UI.Controls/GridSplitter/GridSplitter.Events.cs

```diff
@@ -25,14 +26,26 @@ private void GridSplitter_Loaded(object sender, RoutedEventArgs e)
             _resizeDirection = GetResizeDirection();
             _resizeBehavior = GetResizeBehavior();
 
+            GridSplitterGripper gripper;
+
             // Adding Grip to Grid Splitter
             if (Element == default(UIElement))
             {
-                var element = new GridSplitterGripper(
+                gripper = new GridSplitterGripper(
                     _resizeDirection,
                     GripperForeground);
-                Element = element;
             }
+            else
+            {
+                var content = Element;
+                Element = null;
+                gripper = new GridSplitterGripper(content, _resizeDirection);
+            }
+
+            Element = gripper;
+
+            gripper.OnKeyboardHorizontalMoveAction = (movement) => HorizontalMove(movement);
```

Comment:
What about making a GridSplitter an argument for the ctor of the GridSplitterGripper and have it call the horizontal/vertical move methods rather than callback approach